requirements.txt

```diff
@@ -1,2 +1,3 @@
 flask
 pydantic
+pytest
```

Comment:
I think it's better to add this in the CI part (`.travis.yml` for now) since it's only related to the test part.

Reply:
> I think it's better to add this in the CI part (`.travis.yml` for now) since it's only related to the test part.

I added installation of pytest to `.travis.yml`